plugin/src/test/groovy/com/novoda/test/TestProject.groovy

```diff
@@ -84,15 +84,19 @@ ${project.additionalConfiguration}
         return this
     }
 
-    public T withPlugins(String... plugins) {
-        def formattedPlugins = plugins.collect { plugin ->
-            "id '$plugin'"
-        }.asList()
-
-        this.plugins.addAll(formattedPlugins)
+    public T withPlugin(String plugin, String version = null) {
+        this.plugins.add("id \"$plugin\" ${optionalVersionFrom(version)}")
```

Comment:
`nit:` you can avoid to escape the double quotes if you use simple ones.

Reply:
👍 for single quotes